packages/dev-utils/src/lib/geo-location.test.ts

```diff
@@ -0,0 +1,248 @@
+import { describe, expect, test, vi, beforeEach, afterEach } from 'vitest'
+
+import { getGeoLocation, mockLocation } from './geo-location.js'
+
+// Mock fetch
+global.fetch = vi.fn()
+const mockFetch = vi.mocked(fetch)
+
+describe('geolocation', () => {
+  let mockState: { get: vi.Mock; set: vi.Mock }
```

Comment:
are these the right types? there are type errors below. I think this should be SpyInstance or something?